playbook/app/pb_kits/playbook/pb_icon_circle/_icon_circle.scss

```diff
@@ -21,6 +21,10 @@ $pb_icon_circle_sizes: (
   overflow: hidden;
   position: relative;
 
+  .icon_circle_emoji {
+    font-family: sans-serif;
```

Comment:
This was a tricky one. I noticed that the emojis were not aligning correctly, they were off by 2ish pixels, even with flex, align, etc. After some research I found that different fonts render emojis differently and some fonts can cause misalignment. sans-serif seemed to be the consensus on most basic font that would render emojis without misalignment and it did in fact fix the issue.